### PR TITLE
Fix move plugin issue

### PR DESCRIPTION
**Pull request by mkoistinen**
This issue here is not newsblog, but rather aldryn-reversion, which has now been updated and released. This PR updates the requirements.

### CODE REVIEW

File: aldryn_newsblog/models.py [hunk 1]
```diff
@@ -77,8 +77,7 @@ class Article(TranslatableModel):
     )
 
     content = PlaceholderField('newsblog_article_content',
-                               related_name='newsblog_article_content',
-                               unique=True)
```
Comment (intellectronica) [inline aldryn_newsblog/models.py]: How is removing `unique` related to this change?

Comment (mkoistinen) [inline aldryn_newsblog/models.py]: It was something that shouldn't be there anyway, but it is benign.

Comment (mkoistinen) [inline aldryn_newsblog/models.py]: I'll make separate PRs

Comment (intellectronica) [inline aldryn_newsblog/models.py]: No need to, was just curious.

**Pull request merged**